### PR TITLE
add script for restore and testing

### PR DESCRIPTION
**Pull request by barnabasJ**

### CODE REVIEW

File: VERSION.txt [hunk 1]
```diff
@@ -1 +1 @@
-unversioned
\ No newline at end of file
+f761dba.dirty
```
Comment (dobe) [inline VERSION.txt]: this must not be under version control

File: README.md [hunk 2]
```diff
@@ -1,4 +1,79 @@
...
+* use wal-e backup-fetch /pgdata LATEST to restore the latest backup
+* place a recovery.conf file in the /pgdata directory
+* start postgres
+
+To make this all a bit easier a script is located at /scripts/restore.sh
+It needs the RESTORE_COMMAND variable to be set to the correct command to 
+restore the wal files. Which it uses to create the 
+[recovery.conf](https://www.postgresql.org/docs/9.2/continuous-archiving.html) 
+file and place it inside the /pgdata directory. The backup can be specified as 
+the first argument for the script. For example `/scripts/restore.sh LATEST`.
```
Comment (dobe) [inline README.md]: we could probably put this into /usr/local/bin, this is easier for the user.

**Review by dobe — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (barnabasJ) [reply README.md]: good idea, I'll change the Dockerfile + README

File: VERSION.txt (hunk 1 above)
Comment (barnabasJ) [reply VERSION.txt]: I'll add it to .gitignore

**Review by dobe — APPROVED** [review]

**Pull request merged**